Cabal/Distribution/Simple/Compiler.hs

```diff
@@ -339,6 +340,10 @@ libraryDynDirSupported comp = case compilerFlavor comp of
  where
   v = compilerVersion comp
 
+-- | Does this compiler's "ar" command supports @file-style arguments
+arAtFileArgumentSupported :: Compiler -> Bool
```

Comment:
I suggest that we standardise on calling them "response files" in our codebase, since there's existing precedent (`haddockSupportsResponseFiles`).